modules/validator/schema-validator.js

```diff
@@ -23,7 +23,7 @@ class SchemaValidator {
     }
 
     _getSignerAddress(document) {
-        const merkleRoot = importUtilities.calculateDatasetRootHash(document['@graph'], document['@id'], document.datasetHeader.dataCreator);
+        const merkleRoot = importUtilities.calculateDatasetRootHash(document);
```

Comment:
Do we need to call otjsonservice before calcualting root hash

Reply:
OtJsonService is invoked inside importUtilities methods